### DISCUSSION TITLE
Background for each molecule in MolsToGridImage?

### DISCUSSION THREAD

**Discussion by Boilermaker14** [rdkit/rdkit#6248] [Q&A] [answered]
I'd like to indicate a binary feature across a list of molecules drawn with MolsToGridImage. Is there any straightforward way to pass a list of background colors to the function, such that each mol has it's own background?

**Comment by greglandrum** [comment, marked as answer]
@Boilermaker14 There's not currently an easy way to do this, but it wouldn't be hard to add.
Could you please create an "enhancement" issue for this?

**Reply by Boilermaker14** [reply]
Thanks - it's created as issue #6256
